netutils/ip.py

```diff
@@ -221,11 +223,11 @@ def netmask_to_cidr(netmask):
         23
     """
     if is_netmask(netmask):
-        return bin(int(ipaddress.ip_address(netmask))).count("1")
+        return bin(int(ipaddress.ip_address(netmask))).count("1")  # type: ignore
```

Comment:
Should change the return value of the function to `int`, then we can get rid of the ignore comment.